src/main/java/Alice.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+
+public class Alice {
+    private static final String LINE = "____________________________________________________________\n";
+    private static Task[] tasks = new Task[100];
+    private static int numberOfTask = 0;
+
+    /**
+     *
+     * List all tasks that have been added by user.
+     *
+     */
+    public static void listTasks(){
+        for (int i=0; i<numberOfTask; i++){
```

Comment:
What do you think about leaving spaces before and after operators?
```suggestion
        for (int i = 0; i < numberOfTask; i++) {
```